lessons/basics/modules.md

```diff
@@ -194,9 +202,28 @@ import List, only: :functions
 import List, only: :macros
 ```
 
+It is also worth noting that import is lexical in scope.
+
+```elixir
+iex> defmodule MyModule do
+...>   def func_a do
+...>     import List, only: [duplicate: 2]
```

Comment:
I decided to leave this because it is specific to the behaviour of `import` itself, and worth noting. But now that I think more, its also true that `require` and `import` are also lexical. Maybe I should edit to just make one blanket statement about all of them, and not encourage it but just make it known.

Thoughts?